scripts/extract_timings.py

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env python
+# Time-stamp: <2016-05-31 14:58:27 chrplr>
+#
+
+""" Extracts slice timing information from a dicom file.
+To be passed  as argument to slice_order for slice timing correction algorithms """
+
```

Comment:
On Sun, Jun 19, 2016 at 2:47 PM, Christophe Pallier <
notifications@github.com> wrote:

> In scripts/extract_timings.py
> <https://github.com/neurospin/pypreprocess/pull/231#discussion_r67616387>:
>
> > @@ -0,0 +1,39 @@
> > +#! /usr/bin/env python
> > +# Time-stamp: <2016-05-31 14:58:27 chrplr>
> > +#
> > +
> > +""" Extracts slice timing information from a dicom file.
> > +To be passed  as argument to slice_order for slice timing correction algorithms """
> > +
>
> Hi, I have put a dataset at:
> https://www.nitrc.org/frs/download.php/9035/epi-example-dicom.tar.gz As
> soon as spm12 works in the master branch of pypreprocess,
>
I've merged a PR which should adress the issue that was raised on spm12 and
mcr stuff, at least in part. My bet is spm12 works now, but it'd be great
if someone with spm12 could confirm.

> I will test it on this dataset. I still need to write code to fetch the
> dataset... Chris
> … <#m_6929222270908548504_>
> -- Christophe Pallier Contact info:
> http://www.pallier.org/w/pmwiki.php/Main/Contact
> On Sun, Jun 19, 2016 at 1:18 PM, DOHMATOB Elvis ***@***.***> wrote: In
> scripts/extract_timings.py <#231 (comment)
> <https://github.com/neurospin/pypreprocess/pull/231#discussion_r67615123>>:
> > @@ -0,0 +1,39 @@ > +#! /usr/bin/env python > +# Time-stamp: <2016-05-31
> 14:58:27 chrplr> > +# > + > +""" Extracts slice timing information from a
> dicom file. > +To be passed as argument to slice_order for slice timing
> correction algorithms """ > + If I remember good, we were considering
> hosting a small dataset (say on NTRC), that could be used here to make this
> demo self-contained. Any update ? — You are receiving this because you were
> mentioned. Reply to this email directly, view it on GitHub <
> https://github.com/neurospin/pypreprocess/pull/231/files/8caa896e8ea92a2478829bd93dbdbe53d287f26a#r67615123>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ABdbMvWtLial8myl397edxDu0Il1GHwSks5qNSWAgaJpZM4IqgU1>
> .
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/neurospin/pypreprocess/pull/231/files/8caa896e8ea92a2478829bd93dbdbe53d287f26a#r67616387>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAms1CVDE8NveGHlAfF1MMe3rnDqVnrXks5qNTpVgaJpZM4IqgU1>
> .
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob